flutter/lib/backend/bridge/run_result.dart

```diff
@@ -1,6 +1,6 @@
 import 'package:mlperfbench_common/data/results/benchmark_result.dart';
 
-class RunResult {
+class NativeRunResult {
```

Comment:
I think adding a brief description for each of the class as a comment will be helpful since the class name some time is not descriptive enough.